source/configuration_manual/lazy_expunge_plugin.rst

```diff
@@ -174,17 +169,11 @@ IMAP COPY command to copy the message to Trash before expunging the message
 from the original mailbox.  Deleting later from Trash would result in two
 copies of the same message in the lazy expunge storage.
 
-With v2.2+ you can set ``lazy_expunge_only_last_instance = yes`` to copy only
-the last instance to the expunge storage.  This ensures that only a single
+With v2.2+ you can enable
+:ref:`plugin-lazy-expunge-setting-lazy_expunge_only_last_instance` to copy
+only the last instance to the expunge storage.  This ensures that only a single
 copy of a message will appear in the expunge storage.
 
-This setting works with the following mailbox formats:
```

Comment:
Although this was moved to settings page, maybe there could be a note here also "Note that this setting doesn't work with all mailbox storages."